litestar/pagination.py

```diff
@@ -28,7 +29,7 @@ class ClassicPagination(Generic[T]):
 
     __slots__ = ("items", "page_size", "current_page", "total_pages")
 
-    items: list[T]
+    items: List[T]
```

Comment:
?

Reply:
we need to inspect the types at runtime - python 3.8 is our lowest supported version and `list` is not subscriptable in python 3.8.